### ISSUE TITLE
nimfix: ../lookups.nim(239, 35) Error: type mismatch

### ISSUE THREAD

**Issue by stefantalpalaru** [nim-lang/Nim#9087]
Error appearing in 0.19.0:
```
Hint: linter [Processing]
../lookups.nim(239, 35) Error: type mismatch: got <string>
but expected one of: 
proc getIdent(ic: IdentCache; identifier: string): PIdent
proc getIdent(ic: IdentCache; identifier: string; h: Hash): PIdent
proc getIdent(ic: IdentCache; identifier: cstring; length: int; h: Hash): PIdent

expression: getIdent(toLowerAscii(x.s[0]) & substr(x.s, 1))
```

**Comment by krux02** [comment]
Well nimfix is dead. And there isn't even a question or description in the issue, so I am closing this.

**Comment by krux02** [comment]
Just read the conversation in the linked PR (https://github.com/nim-lang/Nim/pull/9094). Removing it is a good point though.

**Comment by stefantalpalaru** [comment]
A non-compiling `nimfix` is still being shipped with Nim-1.6.0:

```text
/var/tmp/portage/dev-lang/nim-1.6.0-r100/work/nim-1.6.0/lib/std/jsonutils.nim(424, 33) Error: undeclared identifier: 'Option'
candidates (edit distance, scope distance); see '--spellSuggest': 
 (2, 1): 'options' [module declared in /var/tmp/portage/dev-lang/nim-1.6.0-r100/work/nim-1.6.0/lib/std/jsonutils.nim(19, 47)]
 (2, 3): 'TOption' [type declared in /var/tmp/portage/dev-lang/nim-1.6.0-r100/work/nim-1.6.0/compiler/options.nim(28, 3)]
```
